docs/source/markdown/options/creds.md

```diff
@@ -0,0 +1,5 @@
+#### **--creds**=*[username[:password]]*
+
+The [username[:password]] to use to authenticate with the registry if required.
+If one or both values are not supplied, a command line prompt will appear and the
+value can be entered.  The password is entered without echo.
```

Comment:
Tiniest of nits: there's two spaces here, and add a comma while we're at it. Other than that, LGTM
```suggestion
The [username[:password]] to use to authenticate with the registry, if required.
If one or both values are not supplied, a command line prompt will appear and the
value can be entered. The password is entered without echo.
```

Reply:
@ashley-cui thanks, I agree, but I'm not willing to make that change on this or any of my cleanup PRs: they would make review even harder than it already is. Once the refactoring is done, I'm sure that @TomSweeneyRedHat will make a pass with comma-pronoun-space changes. But right now, no, sorry.

Reply:
All good, sorry for the noise!

Reply:
Not noise - it's good feedback, and I will preserve it for a future cleanup PR, thank you!